rebasehelper/plugins/checkers/abipkgdiff.py

```diff
@@ -97,10 +97,11 @@ def run_check(cls, results_dir, **kwargs):
         cls.prepare_results_dir()
         debug_old, rest_pkgs_old = cls._get_packages_for_abipkgdiff(results_store.get_build('old'))
         debug_new, rest_pkgs_new = cls._get_packages_for_abipkgdiff(results_store.get_build('new'))
-        cmd = [cls.CMD]
         ret_codes = {}
+        breakpoint()
```

Comment:
You forgot to remove a breakpoint ;)

Reply:
Right, and not for the first time today :)